wled00/wled.h

```diff
@@ -24,10 +24,18 @@
 //#define WLED_DISABLE_OTA         // saves 14kb
 
 // You can choose some of these features to disable:
-//#define WLED_DISABLE_ALEXA       // saves 11kb
-//#define WLED_DISABLE_BLYNK       // saves 6kb
-//#define WLED_DISABLE_HUESYNC     // saves 4kb
-//#define WLED_DISABLE_INFRARED    // saves 12kb
+#ifndef WLED_ENABLE_ALEXA
```

Comment:
These changes are unnecessary.
Features like Alexa, Blynk, Hue, etc. should be compiled by default and only excluded in customised build environment by defining DISABLE defines.

Reply:
ok, I removed the code (I did it because in SR branch all of these features are disabled)